examples/tracer-web/README.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument your JavScript code running in the browser.
```

Comment:
Minor: JavScript -> JavaScript

Reply:
Fixed.

Reply:
I think we should highlight `@opentelemetry/web` instead of `@opentelemetry/tracing`. 

`@opentelemetry/tracing` is mostly used for the manual instrumentation, whereas `@opentelemetry/web`  module provides automated instrumentation and tracing for Web applications.

Reply:
Updated.